Sources/SwiftSyntax/generated/syntaxNodes/SyntaxDeclNodes.swift

```diff
@@ -71,7 +71,7 @@ public struct AccessorDeclSyntax: DeclSyntaxProtocol, SyntaxHashable {
             unexpectedBetweenInitEffectsAndBody, 
             body, 
             unexpectedAfterBody
-          ))) { (arena, _) in
+          ))) {(arena, _) in
```

Comment:
> Hmm... maybe my local CodeGen did not resolve the HEAD correctly. Will try tomorrow again

I run into this as well @kimdv but apparently @ahoppen does not 🤷. I just delete `.build` and `Package.resolved` each time.

Reply:
I normally also run `swift package update` and `swift package resolve` each time. 
But forgot it 😅

But thanks for the tip!

Should we file a bug in spm repo? 
It would be nice if it checked that automatically.

Reply:
Does `swift package update` and `swift package resolve` always fix the issue for you? Recently I didn’t have the feeling but I think I only ever ran one of the two commands, not both.

I think it’s expected that you need to run `swift package update`. That matches the SwiftPM behavior when you depend on branches where you also only want to update your dependencies (and thus Package.resolved) when you explicitly request it. We should add that to our documentation.

If `swift package resolve` is also necessary, I would say that’s a but in SwiftPM. If you could file an issue for that, that’d be great.

Reply:
Hmmm. Just had a look at the docs, maybe `swift package resolve` is what we should be running? `swift package update` will update `Package.resolved` with the new hash and then `swift package resolve` would update the packages to match that version:
> The swift package resolve command resolves the dependencies, taking into account the current version restrictions in the Package.swift manifest and Package.resolved resolved versions file, and issuing an error if the graph cannot be resolved. For packages which have previously resolved versions recorded in the Package.resolved file, the resolve command will resolve to those versions as long as they are still eligible. If the resolved version's file changes (e.g., because a teammate pushed a new version of the file) the next resolve command will update packages to match that file. After a successful resolve command, the checked out versions of all dependencies and the versions recorded in the resolved versions file will match. In most cases the resolve command will perform no changes unless the Package.swift manifest or Package.resolved file have changed.

But having said that:
> Most SwiftPM commands will implicitly invoke the swift package resolve functionality before running, and will cancel with an error if dependencies cannot be resolved.

So maybe it's a bug if we have to run it, unsure.